### PR TITLE
Provides TSLint action

### PR DESCRIPTION
**Pull request by bltavares**
Suggested by @chagasaway

### CODE REVIEW

File: tslint/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FROM node:alpine
+
+LABEL "com.github.actions.name"="mdlint"
+LABEL "com.github.actions.description"="Provides linting for Markdown files"
```
Comment (chagasaway) [inline tslint/Dockerfile]:
```suggestion
LABEL "com.github.actions.description"="Provides linting for TypeScript files"
```

File: tslint/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
+FROM node:alpine
+
+LABEL "com.github.actions.name"="mdlint"
```
Comment (chagasaway) [inline tslint/Dockerfile]:
```suggestion
LABEL "com.github.actions.name"="tslint"
```

**Review by chagasaway — APPROVED** [review]
👏

File: tslint/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,22 @@
+FROM node:alpine
+
+LABEL "com.github.actions.name"="tslint"
+LABEL "com.github.actions.description"="Provides linting for TypeScript files"
+LABEL "com.github.actions.icon"="user-check"
+LABEL "com.github.actions.color"="blue"
```
Comment (bltavares) [inline tslint/Dockerfile]:
```suggestion
LABEL "com.github.actions.color"="purple"
```

**Pull request merged**